codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/RustType.kt

```diff
@@ -466,6 +466,7 @@ class Attribute(val inner: Writable) {
         val DenyMissingDocs = Attribute(deny("missing_docs"))
         val DocHidden = Attribute(doc("hidden"))
         val DocInline = Attribute(doc("inline"))
+        val ShouldPanic = Attribute("should_panic")
```

Comment:
IMO, we should also support the "message to match against" version and encourage its usage:
```suggestion
        val ShouldPanic = Attribute("should_panic")
        fun shouldPanic(messageToMatchAgainst: String): Writable = macroWithArgs("should_panic", writable(str))
```